### PR TITLE
Update the CHANGELOG

### PR DESCRIPTION
**Pull request by aguerra**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/259)

### CODE REVIEW

**Comment by aguerra** [comment]
@drgarcia1986 @arnaldopereira @yagonobre updated.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,34 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.5.0] - Release Date
+### Changed
+- Use gRPC instead of go-swagger
+- Changed project structure to be more k8s like
+- Standardize errors messages format and server logs
```
Comment (arnaldopereira) [inline CHANGELOG.md]: `error messages` 💅

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,34 @@
...
+- 'app create' command doesn't create a welcome deploy anymore
+- [client] Refactored config and bash completion
+- [server] Delete build pod after deploy on success
+
+### Added
+- Unit tests
+- Travis CI
+- 'set-password' command
+- Builtin TLS (thanks gRPC)
+- Versionning by git tag
```
Comment (arnaldopereira) [inline CHANGELOG.md]: `versioning`

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,34 @@
...
+- [server] 'create-super-user' command
+- [server] added so called 'release' command, which allows developers to run a
+  command right after the build was finished
```
Comment (arnaldopereira) [inline CHANGELOG.md]: `is finished`

**Pull request merged**